packages/components/src/StatusLabel/StatusLabel.mdx

```diff
@@ -0,0 +1,108 @@
+---
+name: StatusLabel
+menu: Components
+route: /components/status-label
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { StatusLabel } from ".";
+import { Content } from "../Content";
+
+# Status Label
+
+The `<StatusLabel>` component is a visual component that allows users to quickly
+determine the status of an item (e.g., "Active", "Overdue").
+
+```ts
+import { StatusLabel } from "@jobber/components/StatusLabel";
+```
+
+<Playground>
+  <StatusLabel label="Bob" />
+</Playground>
+
+## Props
+
+<Props of={StatusLabel} />
+
+## Usage Guidelines
+
+StatusLabel is a more opinionated version of
+[InlineLabel](/components/inline-label) and offers only 5 possible status
+representations:
+
+### Success
```

Comment:
You went into some detail for inactive with actual examples (archived, closed) - would there be value for the others that aren't obvious how they are connected to workflow? (ex Success for Paid, Approved - Critical for late things)